src/lib/components/Widget.js

```diff
@@ -67,11 +67,11 @@ class Widget extends React.Component {
             <img src={Logo} alt="SSW Logo" height="60" width="130"></img>
           </a>
         ) : ''}
-          <h1>
+          <h4>
             <a rel="noreferrer" target="_blank" href={`${this.sswUrl}/rules`}>
               Latest Rules
```

Comment:
oops, this was the one that I meant. 
Might need to add a check to make sure that this doesn't link to the same page. (If this is on /rules it shouldn't link to /rules) as per https://www.ssw.com.au/rules/do-you-avoid-linking-a-page-to-itself
![image](https://user-images.githubusercontent.com/57518417/141396179-45b228ee-8231-4cf6-9277-9427b14ed9b9.png)
Figure: Clicking on "latest rules" links to "https://www.ssw.com.au/rules/"